modules/soe_paragraph_stories/templates/stories-paragraph.html.twig

```diff
@@ -8,49 +8,47 @@
  * Available variables:
  *
  * {{ image }}
- * {{ paragraph_content }}
- * {{ cta }}
+ * {{ person_content }}
+ * {{ cta }}
+ * {{ quote }}
+ * {{ border_color_variant }}
  *
  */
 #}
 {%- set attributes = attributes.addClass(['su-stories-paragraph', modifier_class]) -%}
 
-{%- set img_variant_keys = image|keys -%}
-{%- set img_variant_field_name = img_variant_keys[1] -%}
-{%- set img_variant_field = image[img_variant_field_name] -%}
-{%- set img_variant_value = img_variant_field['#items'][0].value  -%}
-
-{%- if img_variant_value == 1 -%}
-{%- set border_color = 'border-color-1' -%}
-{%- elseif img_variant_value == 2 -%}
-{%- set border_color = 'border-color-2' -%}
-{%- elseif img_variant_value == 3 -%}
-{%- set border_color = 'border-color-3' -%}

```

Comment:
We really don't want to be digging down into specific fields or field values where possible in these templates. I was able to simplify it down by moving the border setting to is own field.